lib/tasks/elasticsearch_reindex.rake

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Executes an ActiveRecord `#touch` call on all ElasticSearch indexed models.
+# Each `#touch` triggers an ElasticSearch reindex with the `:active_job` strategy.
+task elasticsearch_reindex: :environment do
+  task_stats = {}
+  Chewy.strategy(:active_job) do
+    ESIndices.models.each do |model|
+      task_stats[model.to_s] = 0
+      model.find_in_batches(batch_size: 100) do |model_instances|
+        model_instances.each do |model_instance|
+          model_instance.touch # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Will this call to `touch` modify the `updated_at` time on our models? If so, I don't think this is the right approach.

Reply:
It would. 

Another approach would be to read from the db and manually invoke `update_index("model_name#model_name") { model_instance }`. I'll try that approach.